.github/workflows/openqa.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Run a basic openQA test
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+  # Note how this runs on:pull_request_target and not on:pull_request!
+  # The difference is that this runs always with the context of the master
+  # branch. This is necessary to allow accessing the API credential secrets.
+  workflow_dispatch:
+env:
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+  OPENQA_API_KEY: ${{ secrets.OPENQA_API_KEY }}
+  OPENQA_API_SECRET: ${{ secrets.OPENQA_API_SECRET }}
+  GH_REPO: ${{ github.event.pull_request.head.repo.full_name }}
+  GH_REF: ${{ github.event.pull_request.head.ref }}
+
+jobs:
+  trigger_and_monitor_openqa:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/tumbleweed:client
+    steps:
+      - uses: actions/checkout@v3
+      - name: Determine the latest Tumbleweed build on o3
+        id: latest_build
+        run: >-
+          echo build=$(curl -s
+          ${OPENQA_HOST:-https://openqa.opensuse.org}/group_overview/${OPENQA_GROUP_ID:-1}.json
```

Comment:
Where does the `OPENQA_GROUP_ID` come from? I guess in this case it would be tw's group id?

Reply:
Yes. And it defaults to "1", which is the current openSUSE Tumbleweed on o3 but can be overridden with this variable